README.md

```diff
@@ -2,7 +2,7 @@
 
 A Cake Addin for [Squirrel.Windows](https://github.com/Squirrel/Squirrel.Windows).
 
-[![Build status](https://ci.appveyor.com/api/projects/status/vj5f1md7k7jucml7?svg=true)](https://ci.appveyor.com/project/phillipsj/cake-squirrel)
+[![Build status](https://img.shields.io/appveyor/ci/cakecontrib/cake-squirrel.svg?label=appveyor)](https://ci.appveyor.com/project/cakecontrib/cake-squirrel)
```

Comment:
Could you use the following, rather than the `shields.io` link?  Thanks

```suggestion
[![Build status](https://ci.appveyor.com/api/projects/status/6bv8xgvgr5acpdki?svg=true)](https://ci.appveyor.com/project/cakecontrib/cake-squirrel)
```